frontend/packages/console-app/src/redux/reducers/quick-start-reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { QuickStartSidebarActions, Actions } from '../actions/quick-start-actions';
+import { RootState } from '@console/internal/redux';
+
+export const quickStartReducerName = 'quickStart';
+
+type State = {
+  isExpanded: boolean;
+  activeQSID: string;
+};
+
+const initialState: State = {
+  isExpanded: false,
+  activeQSID: '',
+};
+
+export default (state = initialState, action: QuickStartSidebarActions) => {
+  if (action.type === Actions.SetActiveQuickStart) {
+    return {
+      isExpanded: !!action.payload.activeQSID,
+      activeQSID: action.payload.activeQSID || state.activeQSID,
+    };
+  }
+
+  return state;
+};
+
+export const isQuickStartDrawerExpanded = (state: RootState): boolean =>
+  !!state.plugins?.console?.[quickStartReducerName]?.isExpanded;
+
+export const getActiveQSID = (state: RootState): string =>
```

Comment:
```suggestion
export const getActiveQuickStartID = (state: RootState): string =>
```
We aren't abbreviating to qs anywhere else.